src/modules/hero/hero.component.ts

```diff
@@ -15,5 +16,43 @@ export class StacheHeroComponent {
     return this._backgroundImageUrl;
   }
 
+  @Input()
+  set overlayOpacity(opacityPercentage: string) {
+    let validOpacityInput = opacityPercentage.replace(/[^\d.-]/g, '');
+    let validOpacityAsString = this.parseOpacityValueToDecimal(validOpacityInput);
+    this._overlayOpacity = validOpacityAsString;
+  }
+
+  get overlayOpacity(): string {
+    return this._overlayOpacity;
+  }
+
   private _backgroundImageUrl = 'none';
+  private _overlayOpacity = '0.6';
+
+  private parseOpacityValueToDecimal(opacityValue: string): string {
+    let percentageCandiate = parseInt(opacityValue, 10);
+    if (opacityValue.includes('.')) {
+      percentageCandiate = Math.round(parseFloat(opacityValue) * 100) / 100;
+    }
+    let validDecimal = this.validateOpacityDecimal(percentageCandiate);
+    return String(validDecimal);
+  }
+
+  private validateOpacityDecimal(value: number): number {
+    let validDecimal = value;
+    if (value > 100) {
+      validDecimal = 1;
+    } else if (value < 0) {
+      validDecimal = 0;
+    } else if (isNaN(value)) {
+      validDecimal = 0.40;
```

Comment:
This default value is different from the `0.6` set above. Maybe create a `readonly` property for the default?